drivers/mmc/host/sdhci-of-aspeed.c

```diff
@@ -171,6 +171,11 @@ static int aspeed_sdhci_probe(struct platform_device *pdev)
 	dev_info(&pdev->dev, "Configuring for slot %d\n", slot);
 	dev->width_mask = !slot ? ASPEED_SDC_S0MMC8 : ASPEED_SDC_S1MMC8;
 
+	if (of_property_read_bool(pdev->dev.of_node, "aspeed,inv-presence")) {
+		host->quirks2 |= SDHCI_QUIRK2_INVERTED_CARD_DETECTION;
+		dev_info(&pdev->dev, "aspeed: signal presence inversion\n");
+	}
+
```

Comment:
I propose you don't mention the Vesnin in the commit message as it is a global change.

Reply:
I second that. Missed that during my review. This commit definitely doesn't need to mention vesnin.
It must be "... in case of inverted signalling in board schematics".

Reply:
`add inversion signal presence quirk for vesnin` must also be `add a quirk for inverted card detection`